app/views/tournaments/edit.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="container">
+  <h1>Manage <%= @tournament.event.location %></h1>
+  <div class="d-flex justify-content-between mb-4">
+    <div class="">
+      <%= simple_form_for @tournament do |f| %>
+        <%= f.input :pairing_system, collection: Tournament.pairing_systems.keys, include_blank: false %>
+        <%= f.input :number_of_winners %>
+        <%= f.button :submit, class: "btn btn-outline-primary w-100", input_html: { data: { controller: 'start-tournament' } } %>
+      <% end %>
+    </div>
+
+    <div class="col-6">
+      <ul>
+        <% if @tournament.players.nil? %>
+          <div>No players added yet in Pairwise</div>
+        <% else %>
+          <% @tournament.players.each do |player| %>
```

Comment:
Tournament doesn't have players as a column right? I think it's @tournament.event.number_of_players

Reply:
a tournament uses the has_many association to connect with players